### ISSUE TITLE
Example doesn't render partials

### ISSUE THREAD

**Issue by itsthesteve** [grow/grow-ext-kintaro#16]
The example doesn't render the partial templates.  Solution via Jeremy:

```
...
{% set basename = doc.collection.schemas_to_partials.get(field.schema_name) %}
{# If the schema name is not set, skip. #}
{% if not basename %}
  {% continue %}
{% endif %}

{# Determine whether the field corresponds to a partial. #}
{% set partial_filename = "/views/partials/{}.html".format(basename) %}
...

**Comment by Zoramite** [comment]
The example is actual correct. But the preferred naming of the partials in the schema has changed from what the example was using. As long as the schema is in the format: `Partial<name>` then it will be recognized as a partial and the filter for the filename will work correctly. You can also adjust this with the filter to make it work with other prefixes:

`{% set partial_filename = field.schema_name|kintaro.schema_name_to_partial(prefix='section') %}`

I'm going to update the schema for the example to have the correct naming then the example should work correctly.

**Comment by Zoramite** [comment]
Fixed by 0856c51fcc6449deca726391a2ccfb53335b6e68
